packages/config-manager/src/index.js

```diff
@@ -1,191 +1,10 @@
 const AWS = require('aws-sdk')
 const fs = require('fs')
 
-const { generateConfigTsDef } = require('./utils/generate-config-ts-def')
```

Comment:
We can't delete config manager code am afraid - it's a public NPM package, needs to be kept alive

Reply:
We also document it as a way of storing secrets

Reply:
@willmcvay So could you describe more for me. Because we'll not use reapit-config.json anymore. So ts-config will be scaffold by react scaffolder and in the global.d.ts

Reply:
@willmcvay so i'll revert config-manager and just add fetch config for the new one function